File: beecrowd/javascript/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [a,b,c] = lines.shift().split(" ").map((x)=> Number(x))

let pi =  3.14159;
let triangulo = a*c/2;  
let circulo =pi*c*c;
let trapezio = ((a+b)*c)/2;
let quadrado = b*b;
let retangulo = a*b;

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`)
console.log(`CIRCULO: ${circulo.toFixed(3)}`)
console.log(`TRAPEZIO: ${trapezio.toFixed(3)}`)
console.log(`QUADRADO: ${quadrado.toFixed(3)}`)
console.log(`RETANGULO: ${retangulo.toFixed(3)}`)

File: beecrowd/python/1012.py
a = input().split()
#3.0 4.0 5.2
#print("%0.3f"%trap)
pi = 3.14159

A = float(a[0])
B = float(a[1])
C = float(a[2])


trian = (A*C)/2
circ = pi*(C*C)
trap = ((A+B)*C)/2
quadr = B*B
reta = A*B

print("TRIANGULO: %0.3f"%trian)
print("CIRCULO: %0.3f"%circ)
print("TRAPEZIO: %0.3f"%trap)
print("QUADRADO: %0.3f"%quadr)
print("RETANGULO: %0.3f"%reta)
